### DISCUSSION TITLE
How should I set the dataset_size in init_reaction?

### DISCUSSION THREAD

**Discussion by nnyyxx-bit** [deepmodeling/dpgen#1419] [Q&A]
How should I set the dataset_size in init_reaction? 
The combustion simulation system contains NH3, CH4, O2. 
600 trajectories will be simulated at each temperature if set to 100 according to the template. 
As the number of simulation steps increases, it will take a long time. Also, the simulation time will increase exponentially at multiple temperatures.
The predicted time is much longer than the two weeks stated in the article.(8 NVIDIA V100 GPU cards)
Article: ZENG J, ZHANG L, WANG H, et al. Exploring the Chemical Space of Linear Alkane Pyrolysis via Deep Potential GENerator [J]. Energy & Fuels, 2021, 35(1): 762-9.
![屏幕截图 2023-12-05 105837](https://github.com/deepmodeling/dpgen/assets/144452397/78a141d0-2b81-4b94-9337-354530445ff4)
![屏幕截图 2023-12-05 112104](https://github.com/deepmodeling/dpgen/assets/144452397/e66f099e-602e-458a-a9f0-2a32c11e2a30)

**Comment by njzjz** [comment]
I don't understand your meaning. Why are there 600 trajectories?

**Reply by nnyyxx-bit** [reply]
I may have misrepresented it. It should be the "initial configurations" you mentioned in your article. 
![屏幕截图 2023-12-13 180527](https://github.com/deepmodeling/dpgen/assets/144452397/5460543b-7c48-4c78-8771-748d467babe3)
When I set "dataset_size": 100, it will generate more than 600 initial configurations in 01.build folder. 
The number of initial configurations is probably related to the reaction system.
![屏幕截图 2023-12-13 175238](https://github.com/deepmodeling/dpgen/assets/144452397/eef5140d-de18-484a-8931-39633fdf8184)
When model deviation is performed, the MD of each iteration starts with these 600+ initial configurations.
So what I want to ask is how to set the dataset_size to produce the right number of initial configurations, which can balance the efficiency and the quality of the dataset.

**Reply by njzjz** [reply]
The `*.gjf` files you showed are the initial data set. The initial configuration is `data.hydrogen`.
